### PR TITLE
Unauthenticated User login to view account page

### PR DESCRIPTION
**Pull request by indiesquidge**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
```
Comment (indiesquidge) [inline app/controllers/sessions_controller.rb]: Uhhh....what?

Comment (indiesquidge) [inline app/controllers/sessions_controller.rb]: Uhh....what?

Comment (mirjoy) [inline app/controllers/sessions_controller.rb]: According to stack overflow "What that warning is saying is that, according to rubocop's standard, you have too many assignments, branches, and conditionals (ABCs) in that method. "

That said, it looks fine to me. We haven't learned a cleaner way to do that, and you need to keep the params in the controller.

Comment (mirjoy) [inline app/controllers/sessions_controller.rb]: That being said, line 9 could get abstracted to a method like so:

def log_in
  session[:user_id] = @user.id
end

File: spec/users/user_manages_cart_spec.rb [hunk 2]
```diff
@@ -31,7 +31,6 @@
     click_link_or_button "Add to Cart"
     click_link_or_button "Add to Cart"
     visit "/cart"
-    save_and_open_page
```
Comment (mirjoy) [inline spec/users/user_manages_cart_spec.rb]: delete this line

Comment (indiesquidge) [inline spec/users/user_manages_cart_spec.rb]: What do you mean? That line is deleted.

**Comment by teresafinn** [comment]
Close #31

File: spec/users/user_manages_cart_spec.rb (hunk 2 above)
Comment (mirjoy) [inline spec/users/user_manages_cart_spec.rb]: my bad...long day

**Pull request merged**